nextflow.config

```diff
@@ -69,7 +69,7 @@ params {
     save_cat_db                 = false
 
     // Eukulele options
-    eukulele_db                 = null
+    eukulele_db                 = ''
```

Comment:
Sorry for my ignorance, but why `''` and not `null`? And why then `null` on `dbname`?

Reply:
I noticed that `null` it actually doesn't print "null" so I changed. (I got this error while testing this issue).